README.md

```diff
@@ -212,15 +201,15 @@ VCR.configure do |config|
 
   # when using agentless mode
   # note to use the correct datadog site (e.g. datadoghq.eu, etc)
-  config.ignore_hosts "citestcycle-intake.datadoghq.com", "api.datadoghq.com"
+  config.ignore_hosts "citestcycle-intake.datadoghq.com", "api.datadoghq.com", "citestcov-intake.datadoghq.com"
```

Comment:
Just out of curiosity, how does this interact with our staging endpoints?

Reply:
Our users do not hit staging endpoints so there is no need to include it in the docs. When I use open source porject that uses VCR, I add all the endpoints I need to ignore in project's configs.